btrfs_tree.go

```diff
@@ -0,0 +1,315 @@
+/*
+   Copyright The containerd Authors.
+
```

Comment:
Should the headers in this project be updated to preserve the old copyright? Or do we consider "The containerd authors" to cover this?

```suggestion
   Copyright The containerd Authors.
   Copyright 2016-2021 Denys Smirnov.
```

Reply:
That will not pass the CI 😅

Reply:
Oh 🤦 we use a single set of templates for all project 😞